app/helpers/application_helper.rb

```diff
@@ -1,8 +1,32 @@
 module ApplicationHelper
 
-def redirect_back_or(default)
-  redirect_to(session[:return_to] || default)
-  session.delete(:return_to)
-end
+	def redirect_back_or(default)
+		  redirect_to(session[:return_to] || default)
+		  session.delete(:return_to)
+	end
+
+	# calculate total entitlement for current period
+	def current_holiday_entitlement_id(staff)
+		@holiday_allowances = 0
+		@staff = staff
+		@date_now = Time.zone.now
+		@holiday_allowances = HolidayEntitlement.where(:staff_id => @staff.id).where("holiday_year_start <= ?", @date_now).where("holiday_year_end >= ?", @date_now).first
```

Comment:
Line is too long. [164/80]<br>Use the new Ruby 1.9 hash syntax.<br>Tab detected.